scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl

```diff
@@ -161,62 +169,6 @@ workflow GvsCalculatePrecisionAndSensitivity {
   }
 }
 
-task IsVcfOnChromosomes {
-  input {
-    File input_vcf_fofn
-    Int index
-    Array[String] chromosomes
-    String variants_docker
-  }
-
-  command <<<
-    # Prepend date, time and pwd to xtrace log entries.
-    PS4='\D{+%F %T} \w $ '
-    set -o errexit -o nounset -o pipefail -o xtrace
-
-    # Get a one-based line number from the zero-based input index.
-    line_number=$((~{index} + 1))
-    input_vcf_path=$(sed -n "${line_number}p" ~{input_vcf_fofn})
-
-    input_vcf=$(basename ${input_vcf_path})
-    gcloud storage cp ${input_vcf_path} ${input_vcf}
-    cat ${input_vcf} | gunzip | grep -v '^#' | cut -f 1 | sort | uniq > chrom.txt
-    NL=$(cat chrom.txt | wc -l)
-    if [ $NL -ne 1 ]; then
-      echo "${input_vcf_path} has either no records or records on multiple chromosomes"
-      exit 1
-    fi
-
-    mkdir output
-    output_vcf_name=${input_vcf}
-    touch output/${output_vcf_name}
-    VCF_CHR=$(cat chrom.txt)
-    chromosomes=( ~{sep=' ' chromosomes} )
-
-    for i in "${chromosomes[@]}"
-    do
-      if [ $VCF_CHR = $i ]; then
-        cp ${input_vcf} output/${output_vcf_name}
-        echo "Including ${input_vcf_path} as it is on $i."
-        break
-      else
-        touch output/${output_vcf_name}
-        echo "Skipping ${input_vcf_path} as it is not on $i."
-      fi
-    done
-  >>>
```

Comment:
omg why was this a thing lol

Reply:
This was part of @gbggrant's work for VS-390